app/src/main/java/com/sci/recipeandroid/feature/personalize/ui/adapter/DietRecipeAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.sci.recipeandroid.feature.personalize.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sci.recipeandroid.databinding.ItemViewDietRecipeBinding
+import com.sci.recipeandroid.feature.personalize.domain.model.DietRecipeModel
+import com.sci.recipeandroid.feature.personalize.ui.viewholder.DietRecipeViewHolder
+
+class DietRecipeAdapter(private val onClickItem:(Int)->Unit): RecyclerView.Adapter<DietRecipeViewHolder>() {
+    private var List = emptyList<DietRecipeModel>()
```

Comment:
not "List" , "list" should be

Reply:
Update in e2ceaf1519d2ad526bf9d419752063a820c4ad13